hpobench/container/benchmarks/surrogates/yahpo_gym.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+""" Benchmark for the YAHPO surrogates Benchmark from hpobench/benchmarks/surrogates/yahpo_gym.py
```

Comment:
Do we still need that?

Reply:
I mean the line5. that also creates a codestyle error. 
Could you please add a line break.